static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -119,10 +121,11 @@ const isCompatibleWithEpic = (page: Object): boolean =>
     isArticleWorthAnEpicImpression(page, defaultExclusionRules);
 
 const shouldShowReaderRevenue = (
-    showToContributorsAndSupporters: boolean = false
```

Comment:
The logic here is subtly different so that we can now avoid having to filter out non-supporters in the `canRun` function.
Previously, even if we passed in `true` here, the function would return `true` if the user is a non-supporter.